libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFSalesforceAction.m

```diff
@@ -35,6 +35,11 @@
 
 static void * kObservingKey = &kObservingKey;
 
+@interface CSFNetwork() <SFUserAccountManagerDelegate> {
```

Comment:
I had to move this here as well (seemed like the right place anyway), for user switching to work properly.